cfme/common/__init__.py

```diff
@@ -349,7 +349,11 @@ def get_tags(self, tenant="My Company Tags"):
         """
         view = navigate_to(self, 'Details')
         tags = []
-        tag_table = view.entities.smart_management
+        entities = view.entities
+        if getattr(entities, 'smart_management', False):
+            tag_table = entities.smart_management
+        else:
+            tag_table = entities.summary('Smart Management')
```

Comment:
**REQUIRED**
```python
if hasattr(entities, 'smart_management', False):
    'smart_management', False):
else:
    tag_table = entities.summary('Smart Management')
```

Reply:
It will go, when `get_detail` will be removed from hosts and providers.

Reply:
There's no third option for `hasattr()`, its wrapping `getattr()` and returning false, so just 
```
if hasattr(entities, 'smart_maangement')